report_template.tex

```diff
@@ -134,6 +134,43 @@ \section{Results and Discussion}
   \label{fig:samplefigure}
 \end{figure}
 
+If you have multiple figures that need grouping together, then you may use figures like Figure~\ref{fig:subcaptionfigure}, and each subfigure can be referenced as Figure~\ref{fig:1b}
+
+\begin{figure}
+	\begin{subfigure}[b]{.33\textwidth}
+		\centering\includegraphics[width=\textwidth]{modelfigure_0.pdf}
+		\caption{noise factor = 1.0}\label{fig:1a}
+	\end{subfigure}\hfill
+	\begin{subfigure}[b]{.33\textwidth}
+		\centering\includegraphics[width=\textwidth]{modelfigure_1.pdf}
```

Comment:
Probably better practice to encode the value of the noise factor in the filename somehow

Reply:
Encoded using the value of the noise factor instead of the enumerated index. I've replaced the decimal point with a "p" to ensure there are no filename extension issues. Originally I converted the base 10 decimal to floating-point hexadecimal but at 32 bits long the filename was a little ridiculous.

Reply:
Agreed about the 32 bits!